src/qz/printer/action/PrintRaw.java

```diff
@@ -146,9 +148,10 @@ public void parseData(JSONArray printData, PrintOptions options) throws JSONExce
         }
     }
 
-    private ImageWrapper getImageWrapper(String data, JSONObject opt, boolean fromFile) throws IOException {
+    private ImageWrapper getImageWrapper(String data, JSONObject opt, PrintingUtilities.Flavor flavor, PrintOptions.Pixel pxlOpts) throws IOException {
         BufferedImage bi;
 
+        boolean fromFile = flavor != PrintingUtilities.Flavor.BASE64;
```

Comment:
Why does this still use a fromFile variable instead of directly checking the flavor in the if-statement like getpdfwrapper now does?

Reply:
Because of the legacy variable which is only used for images.  Do you have a better recommendation?

Reply:
explicitly set the flavor to base64 in the compat if-block?

Reply:
Done.